content/en/docs/v3.5/op-guide/configuration.md

```diff
@@ -53,7 +53,9 @@ The list of flags provided below may not be up-to-date due to ongoing developmen
 --listen-peer-urls 'http://localhost:2380'
   List of URLs to listen on for peer traffic.
 --listen-client-urls 'http://localhost:2379'
-  List of URLs to listen on for client traffic.
+  List of URLs to listen on for client grpc traffic and http as long as --listen-client-http-urls is not specified.
```

Comment:
We also need to include this flag in the `v3.4` and `v3.6` configuration pages.

Reply:
Sure, will do